libkpod/container_data.go

```diff
@@ -187,16 +186,16 @@ func getBlankSpec() specs.Spec {
 }
 
 // get an oci.Container instance for a given container ID
-func getOCIContainer(store storage.Store, container string) (*oci.Container, error) {
-	ctr, err := FindContainer(store, container)
+func (c *ContainerServer) getOCIContainer(container string) (*oci.Container, error) {
```

Comment:
I commented on this above before I saw your review.  Yes, we can.  There is the issue of not being able to locate containers not in memory though